python/cac_tripplanner/destinations/views.py

```diff
@@ -375,9 +376,13 @@ def get(self, request, *args, **kwargs):
             for poly in json_poly['features']:
                 geom_str = json.dumps(poly['geometry'])
                 geom = GEOSGeometry(geom_str, srid=4326)
-                matched_objects = (Destination.objects.filter(published=True, point__within=geom)
-                                                      .annotate(distance=Distance('point', geom))
-                                                      .order_by('distance', 'priority'))
+                # include destinations that are published, in a tour, or in an event
+                matched_objects = (Destination.objects.filter(
+                    Q(point__within=geom) &
+                    (Q(published=True) | Q(tours__isnull=False) | Q(events__isnull=False)))
```

Comment:
Yes, this is so unpublished destinations can be used so that places that only apply to events or tours can be added to the app without them showing as stand-alone places. I think you're right it would pull in places on unpublished tours and events; I'll take a look.